modules/gapi/include/opencv2/gapi/streaming/cap.hpp

```diff
@@ -103,6 +103,12 @@ class GCaptureSource: public IStreamSource
     }
 };
 
+// NB: Overload for using from python
+GAPI_EXPORTS_W IStreamSourcePtr inline make_capture_src(const String& path)
+{
+    return make_src<GCaptureSource>(path);
+}
```

Comment:
Why not create a static method `GCaptureSource::create`?

Reply:
To keep consistency between c++ and python code